lib/Parser/CaseInsensitive.cpp

```diff
@@ -579,9 +579,18 @@ END {
         static const int numTransforms = sizeof(transforms) / sizeof(Transform);
         static const Transform lastTransform = transforms[numTransforms - 1];
 
+        static constexpr bool isTransformEntryValid(int offset)
+        {
+            return offset >= numTransforms || ((transforms[offset].delta0 != transforms[offset].delta1 || transforms[offset].delta0 != transforms[offset].delta2 || transforms[offset].delta0 != transforms[offset].delta3) && isTransformEntryValid(offset + 1));
+        }
+
         template <typename Char, typename Fn>
         bool RangeToEquivClass(uint& tblidx, uint l, uint h, uint& acth, Char equivl[EquivClassSize], Fn acceptSource)
         {
+            // Note: There's a few places where we assume that there's no equivalence set
+            // with only one actual member. If you fail this check, double-check the data
+            // in the table above - one line likely has the same value for all deltas.
+            static_assert(isTransformEntryValid(0), "Invalid Transform code - check for 4 identical deltas!");
```

Comment:
Add comment about the 0 base case for static recursive call.

Reply:
done.